### PR TITLE
Adding ability to configure columns for data tables

### PR DESCRIPTION
**Pull request by jhoac**
## **Description**

INSERT A SMALL DESCRIPTION OF WHAT YOU CHANGED HERE

#### **Verify that...**

- [ ] Any related demos were added and `npm start` and `npm run build` still works
- [ ] New demos work in `Safari`, `Chrome` and `Firefox`
- [ ] `npm run lint` passes
- [ ] `npm test` passes and code coverage is increased
- [ ] `npm run build` still works

#### **Bullhorn Internal Developers**
- [ ] Run `Novo Automation`
- [ ] Run `BBO Automation`

##### **Screenshots**

### CODE REVIEW

File: projects/novo-elements/src/elements/data-table/configure-columns/configure-columns-dropdown.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    OnChanges,
+    Output,
+} from '@angular/core';
+// APP
+import { BasePickerResults } from '../../picker/extras/base-picker-results/BasePickerResults';
+
+@Component({
+    selector: 'configure-columns-dropdown',
+    templateUrl: './configure-columns-dropdown.html',
+})
```
Comment (bvkimball) [inline projects/novo-elements/src/elements/data-table/configure-columns/configure-columns-dropdown.component.ts:17]: can you move the styles in here.
`styleUrls: ['./configure-columns-dropdown.scss']`. and use `:host {`

File: projects/novo-elements/src/elements/data-table/configure-columns/configure-columns-dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.dropdown-container.configure-columns-dropdown {
```
Comment (bvkimball) [inline projects/novo-elements/src/elements/data-table/configure-columns/configure-columns-dropdown.scss:1]: use `:host {`

File: projects/novo-elements/src/novo-elements.scss [hunk 3]
```diff
@@ -20,6 +20,7 @@
 // Elements
 // --------------------------------------------------
 @import './elements/data-table/data-table.component';
+@import './elements/data-table/configure-columns/configure-columns-dropdown';
```
Comment (bvkimball) [inline projects/novo-elements/src/novo-elements.scss:23]: remove per adding to component

**Comment by charlesabarnes** [comment]
Closing PR for now, but the branch will remain, feel free to reopen if/when this becomes relevant again

**Pull request closed without merging**